development.cfg

```diff
@@ -1,4 +1,4 @@
 [buildout]
 extends =
-    test-plone-4.3.x.cfg
+    test-plone-5.1.x.cfg
```

Comment:
Should we already switch development to 5.1?
I'd probably rather keep it at 4.3 for now..

Reply:
Good idea. I have changed that back to 4.3.x.